core/conversion/converters/impl/batch_norm.cpp

```diff
@@ -75,8 +82,147 @@ auto batch_norm_registrations TRTORCH_UNUSED = RegisterNodeConversionPatterns().
+           bool use_input_stats, float momentum, float eps, bool cudnn_enabled) -> (Tensor))SIG",
+     [](ConversionCtx* ctx, const torch::jit::Node* n, args& args) -> bool {
+         auto input = args[0].ITensorOrFreeze(ctx); // assumes non-static input Tensor
+         auto orig_shape = input->getDimensions();
+         auto shape = util::toVec(orig_shape);
+         auto tensor_type = util::toATenDType(input->getType());
+         LOG_DEBUG("input type is: "<< tensor_type);
+         auto options = torch::TensorOptions().dtype(tensor_type);
+
+        if(args[3].IValue()->isNone() && args[4].IValue()->isNone())
+        {
+            // Use tensorrt plugin layer when running mean and running variance are not set.
+            Weights gamma, beta;
+         if (ctx->input_is_dynamic) {
+             gamma = Weights(ctx, args[1].unwrapToTensor());
+             beta = Weights(ctx, args[2].unwrapToTensor());
+         } else {
+             gamma = Weights(ctx, args[1].unwrapToTensor(at::full({shape}, 1, {options})));
+             beta = Weights(ctx, args[2].unwrapToTensor(at::full({shape}, 1, {options})));
+         }
+
+         LOG_DEBUG("gamma size: "<< gamma.data.count);
+         LOG_DEBUG("beta size: "<< beta.data.count);
+
+         auto epsilon = args[7].unwrapToDouble(1e-5f);
+
+         LOG_DEBUG("momentum disregarded");
+         LOG_DEBUG("use_input_stats disregarded");
+         LOG_DEBUG("cudnn disregarded");
+
+         auto should_unpack = util::toVec(orig_shape).size() < 4;
+         if (should_unpack) {
+             // expand spatial dims from 1D to 2D
+             auto new_shape = util::toDimsPad(util::toVec(orig_shape), 4);
+             LOG_DEBUG(
+                     "Input shape is less than 4D got: "
+                             << orig_shape << ", inserting shuffle layer to reshape to 4D tensor shape: " << new_shape);
+             auto in_shuffle = ctx->net->addShuffle(*input);
+             in_shuffle->setReshapeDimensions(new_shape);
+             in_shuffle->setName(std::string("[Reshape input to " + util::toStr(new_shape) + ']').c_str());
+             input = in_shuffle->getOutput(0);
+         }
+         std::string pluginName = "InstanceNormalization_TRT";
+         nvinfer1::PluginFieldCollection fc;
+         std::vector<nvinfer1::PluginField> f;
+         f.emplace_back(nvinfer1::PluginField("epsilon", &epsilon, nvinfer1::PluginFieldType::kFLOAT32, 1));
+         f.emplace_back(nvinfer1::PluginField("scales", gamma.data.values, nvinfer1::PluginFieldType::kFLOAT32, gamma.data.count));
+         f.emplace_back(nvinfer1::PluginField("bias", beta.data.values, nvinfer1::PluginFieldType::kFLOAT32, beta.data.count));
+
+         fc.nbFields = f.size();
+         fc.fields = f.data();
+         nvinfer1::IPluginV2* pluginV2 = ctx->mPluginRegistry.at(pluginName)->createPlugin("instancenorm", &fc);
+
+         TRTORCH_CHECK(pluginV2, "Unable to create interpolation plugin from node" << *n);
```

Comment:
interpolation plugin name -> instance norm plugin. To distinguish from line 143 check, this message can be changed to something like "Unable to create instance norm plugin from TensorRT plugin registry"